app/routers/study_dashboard.py

```
@@ -0,0 +1,70 @@
+        user = request.state.user  # 미들웨어에서 설정된 사용자 정보
+        service = StudyDashboardService(db)
+        return await service.get_calendar_study_records(user.id, year, month)
+    except Exception as e:
+        raise HTTPException(
+            status_code=500, detail=f"Failed to fetch calendar study records: {str(e)}"
+        )
+
+
+@router.get("/answer-sheets/in-progress", response_model=InProgressAnswerSheetResponse)
+async def get_in_progress_answer_sheets(
+    request: Request, db: AsyncSession = Depends(get_db)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)